src/main/java/frc/robot/commands/Outtake.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Outtake extends SubsystemBase {
+    /**
+     * Creates a new ExampleSubsystem.
+     */
+    public Outtake() {
+
+    }
+    public static void spin(){
+
+    }
+    public static void stop(){
```

Comment:
What do you mean exactly by stop? like not shoot?

Reply:
yes, i mean like stop spinning

Reply:
i changed it to stopShoot